From 1573b4da4c24e8b3165f4c55991e3dec7484eca8 Mon Sep 17 00:00:00 2001
From: "Partha P. Mukherjee" <ppm.floss@gmail.com>
Date: Mon, 22 Sep 2014 14:19:53 +0530
Description: Fix off-by one error which causes ugly graphs again since 2.4.4
Bug-Debian: https://bugs.debian.org/709533

---
 Chart/Base.pm |    3 +++
 1 file changed, 3 insertions(+)

Index: libchart-perl/Chart/Base.pm
===================================================================
--- libchart-perl.orig/Chart/Base.pm	2015-05-02 09:52:01.559231008 +0200
+++ libchart-perl/Chart/Base.pm	2015-05-02 09:52:01.555231056 +0200
@@ -4459,6 +4459,9 @@
 
         if ( grep { $brushStyle eq $_ } ( 'default', 'circle', 'donut', 'OpenCircle', 'FilledCircle' ) )
         {
+            $xc = $xc - 1;
+            $yc = $yc - 1;
+
             $brush->arc( $xc, $yc, $radius, $radius, 0, 360, $newcolor );
             $brush->fill( $xc, $yc, $newcolor );
 
